src/main/java/org/javarosa/form/api/FormEntryPrompt.java

```diff
@@ -318,7 +292,8 @@ public String getHelpText() {
 
 
     /**
-     * Attempts to return the specified Item (from a select or 1select) text.
+     * Attempts to return the specified item text from a STATIC select or select1. Does NOT
+     * support dynamic selects ({@see ItemsetBinding}).
```

Comment:
This comment suggests I thought this was not applicable to dynamic selects. However, it's now crashing for dynamic selects so something is not right: https://github.com/getodk/javarosa/pull/625#issuecomment-805488046

Reply:
Ah. There's Collect code that directly calls `getSelectItemText` regardless of whether the select is static or dynamic. It seems that used to work and doesn't anymore.